BricksAndHearts.UnitTests/ControllerTests/Admin/AdminControllerTests.cs

```diff
@@ -24,4 +31,37 @@ public void Index_WhenCalledByAnonymousUser_ReturnsViewWithLoginLink()
         result!.ViewData.Model.Should().BeOfType<AdminViewModel>()
             .Which.CurrentUser.Should().BeNull();
     }
+
+    [Fact]
+    public void GetAdminList_ReturnsViewWith_AdminListModel()
+    {
+        // Arrange
+        var fakeAdminService = A.Fake<IAdminService>();
+        var fakeListOfAdmins = A.Dummy<Task<(List<UserDbModel> CurrentAdmins, List<UserDbModel> PendingAdmins)>>();
+        var fakeAdminController = new AdminController(null!, null!, fakeAdminService);
+        A.CallTo(() => fakeAdminService.GetAdminLists()).Returns(fakeListOfAdmins);
+
+        // Act
+        var result = fakeAdminController.GetAdminList().Result as ViewResult;
+        var resultService = fakeAdminService.GetAdminLists();
+
+        // Assert
+        result!.Model.Should().BeOfType<AdminListModel>();
+        resultService.Should().Be(fakeListOfAdmins);

```

Comment:
We don't want to test the service logic in here - this file is for testing the controller logic. Additionally, we have explicitly said that a call to the service will return a specific result, so this doesn't really test anything